frontend/src/pages/pipelines/global/experiments/artifacts/__tests__/ArtifactsTable.spec.tsx

```diff
@@ -103,7 +103,7 @@ describe('ArtifactsTable', () => {
                 ],
               ],
               state: 2,
-              createTimeSinceEpoch: 1611399342384,
+              createTimeSinceEpoch: new Date(),
```

Comment:
This one could be left alone to test the date string format

Reply:
Good point. Reverted this to the original.